src/cljs/witan/ui/controllers/user.cljs

```diff
@@ -19,6 +19,11 @@
   {:username email-regex
    :password password-regex})
 
+(def SignUp
+  (merge Login
```

Comment:
Always seems to be idiomatic for Prismatic Schema objects https://github.com/plumatic/schema#more-examples